internal/server/recon/recognize.go

```diff
@@ -81,6 +86,21 @@ func RecognizePlaces(
 	return resp, nil
 }
 
+// Get a map of reconName values to list of possible types for that entity to
+// be recognized
+func getReconName2RequiredTypes() map[string][]string {
+	typeRequiredEntitiesMap := map[string][]string{}
+	_, filename, _, _ := runtime.Caller(0)
+	bytes, err := os.ReadFile(path.Join(path.Dir(filename), reconName2RequiredTypes))
+	if err == nil {
+		err = json.Unmarshal(bytes, &typeRequiredEntitiesMap)
+		if err != nil {
+			return map[string][]string{}
+		}
+	}
+	return typeRequiredEntitiesMap
+}
```

Comment:
We don't want to read this for every request.   Instead this file should be part of: https://github.com/datacommonsorg/mixer/blob/6e695ef4deb8dcd3f15e9086ffa57195277b794f/internal/store/files/recog_place_store.go#L81

Reply:
done